examples/sftda/01-spin_square.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env/python
+
+'''
+Test the spin_square calculation tool in sftda.tools_td.py.
+'''
+
+from pyscf import gto,dft,sftda,tdscf
+from pyscf.sftda import tools_td
+
+# ToDo : add the spin flip TDDFT parts.
```

Comment:
Can you elaborate on this? Is this function missing terms?

Reply:
Not this function missing terms. There isn't a good davidson or other solvers to solve the spin flip tddft matrix, and the sftda module  can be used in spin flip tda using the davidson solver in pyscf only. For the spin flip tddft, we recommend to solve the whole spin flip tddft matrix by the help of get_ab_sf fucntion, which has been applied in the sftda module,  until we or other people find a good solver. So the sftda module can not give a spin flip tddft calculation results directly, that need users to construct a calculation result to give tools_td.spin_square() and obtain the <S^2>.